token/services/identity/msp/types.go

```diff
@@ -14,3 +14,22 @@ const (
 	// IdemixIdentity identifies an idemix identity
 	IdemixIdentity identity.Type = "idemix"
 )
+
+const (
+	// OwnerMSPID is the default MSP ID for the owner wallet
+	OwnerMSPID = "OwnerMSPID"
```

Comment:
It's better to have a type for MSPIDs. They are mapped 1-1 and they're easy to confuse.